scripting/tf2itemplugin_weapons.sp

```diff
@@ -9,273 +17,260 @@
 #pragma newdecls required
 
 #define PLUGIN_VERSION "4.0.0"
-#define DEBUG		   false
+#define DEBUG          false
 
 public Plugin myinfo =
 {
-	name		= "TF2 Item Plugins - Weapons Manager",
-	author		= "Lucas 'punteroo' Maza",
-	description = "Customize your weapons and manage your server inventory.",
-	version		= PLUGIN_VERSION,
-	url			= "https://github.com/punteroo/TF2-Item-Plugins"
+    name        = "TF2 Item Plugins - Weapons Manager",
+    author      = "Lucas 'punteroo' Maza",
```

Comment:
You can add yourself as a co-author such as `& vexx-sm` if you like.

Reply:
That wouldn't be necessary, Just helping out :)